packages/rpc-core/src/rpc-methods/__tests__/get-block-test.ts

```diff
@@ -0,0 +1,61 @@
+// import { createHttpTransport, createJsonRpc } from '@solana/rpc-transport';
+// import type { Rpc } from '@solana/rpc-transport/dist/types/json-rpc-types';
+import fetchMock from 'jest-fetch-mock-fork';
+// import { createSolanaRpcApi, SolanaRpcMethods } from '../index';
+
+// These tests can't be run reliably since we can't predict where the ledger
+// will be when this suite gets invoked.
+// For example, if you call `getBlock` with slot `0` and the ledger has moved
+// well beyond that, the test will fail because the block will be cleaned up.
```

Comment:
Yeah, this was a problem with the old web3.js tests for sure.

It's not great, but the way that's handled over there is that the tests fetch a fresh slot number using `getSlot()` and then start from there. Though that would mean that we're testing more than one thing, if you relegated that fetch to the `beforeEach()` step would that buy us any benefits in terms of being able to catch breaking changes in a future version of the RPC?